platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init

```diff
@@ -11,6 +11,23 @@
 # Short-Description: Setup Haliburton board.
 ### END INIT INFO
 
+setup_swap () {
+    SWAPFILE=/host/myswapfile
+
+    if [ ! -f $SWAPFILE ]; then
+        availspace=`df -h --output=avail /host | sed '1d;s/[^.0-9]//g'`
+        if awk "BEGIN {exit !($availspace > 2)}"; then
```

Comment:
Why not `[[ $availspace -gt 2 ]]`?

Reply:
because the number could be floating point

Reply:
Let's only compare the integer part, we need to have 2x swap_size avail to make sure we have enough disk space to store logs or other things. Then we can compare >= 4